### PR TITLE
Add Frigate Beta

### PR DESCRIPTION
**Pull request by mtthidoteu**
Allowed selecting beta frigate images. This is useful because frigate has been completely rebuilt

### CODE REVIEW

**Comment by ElectronicBattle** [comment]
I was just about to make the same request please: 0.14 means a new UI for Frigate and numerous bug fixes, hence an *option* to choose 0.13 or 0.14 beta would be tremendous.

**Comment by mtthidoteu** [comment]
> I was just about to make the same request please: 0.14 means a new UI for Frigate and numerous bug fixes, hence an *option* to choose 0.13 or 0.14 beta would be tremendous.

Exactly! My only concern would be the port changes, however 0.14 still has support for the legacy ports so for now having only the proposed 5000 port would be fine

**Comment by ElectronicBattle** [comment]
I note the lines in `/mnt/pool/ix-applications/releases/frigate/charts/1.2.6/ix_values.yaml` which say

```
image:
  repository: ghcr.io/blakeblackshear/frigate
  pullPolicy: IfNotPresent
  tag: 0.13.2
```
and I wonder what would happen if "0.13.2" was changed to "0.14.0"  , or is this yet another of my daft ideas which will cause more trouble than it's worth ...

**Comment by stavros-k** [comment]
Please fix linting errors and also adjust upgrade_strategy
Thanks

File: library/ix-dev/community/frigate/upgrade_strategy [hunk 1]
```diff
@@ -7,6 +7,7 @@ from catalog_update.upgrade_strategy import semantic_versioning
 
 
 RE_STABLE_VERSION_BASE = r'\d+\.\d+\.\d+'
+RE_BETA_VERSION_BASE = r'\d+\.\d+\.\d+-beta\d+'
```
Comment (stavros-k) [inline library/ix-dev/community/frigate/upgrade_strategy:10]: The `-beta` will have to be removed too when getting the version of the semantic versioning.
As it wont be sortable

File: library/ix-dev/community/frigate/upgrade_strategy [hunk 2]
```diff
@@ -38,7 +47,7 @@ def newer_mapping(image_tags):
         if not version:
             continue
 
-        if key == 'image':
+        if key == 'image' or key == 'imageBeta':
```
Comment (stavros-k) [inline library/ix-dev/community/frigate/upgrade_strategy:50]: Please rely on the `image` only for setting the app_version.

File: library/ix-dev/community/frigate/upgrade_strategy [hunk 3]
```diff
@@ -52,4 +61,4 @@ if __name__ == '__main__':
     except ValueError:
         raise ValueError('Invalid json specified')
 
-    print(json.dumps(newer_mapping(versions_json)))
+    print(json.dumps(newer_mapping(versions_json)))
```
Comment (stavros-k) [inline library/ix-dev/community/frigate/upgrade_strategy:64]: Please fix EOL

**Pull request closed without merging**